src/inspector/tracing_agent.cc

```diff
@@ -192,6 +192,8 @@ DispatchResponse TracingAgent::getCategories(
   categories_list->addItem("node.vm.script");
   categories_list->addItem("v8");
   categories_list->addItem("node.http");
+  categories_list->addItem("node.threadpoolwork.sync");
```

Comment:
nit: could we order the list alphabetically?

Reply:
Fixed !

Reply:
Maybe we should also check this in test ?

Reply:
I have update the test.

Reply:
Yes, i will update later.